Ginger/Ginger/SolutionWindows/TreeViewItems/ApplicationModelsTreeItems/ApplicationPOMsTreeItem.cs

```diff
@@ -147,9 +147,21 @@ void ITreeViewItem.SetTools(ITreeView TV)
         }
 
         internal void AddPOM(object sender, RoutedEventArgs e)
-        {           
+        {
+            ObservableList<ApplicationPlatform> TargetApplications = new ObservableList<ApplicationPlatform>();
+            if (WorkSpace.Instance.Solution.ApplicationPlatforms != null)
+            {
+                TargetApplications = GingerCore.General.ConvertListToObservableList(WorkSpace.Instance.Solution.GetListOfPomSupportedPlatform());
```

Comment:
no need to cast it to an observable list. because you just need to check the count!=0